### PR TITLE
Implement basic header react component 

### LINKED ISSUE #37: UI: Create header component
@dnastrain 

Parent issue: #22

### PR DESCRIPTION
**Pull request by dwcares**
Implement basic header component per mockup. Title can be set via props.title.

Fixes #37

@dnastrain 

navbar

### CODE REVIEW

File: src/components/NavBar/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  handleChange (event) {
+    this.setState({value: event.target.value})
+  }
+
+  render () {
+    return (
+      <nav className='NavBar-nav'>
+        <h1 className='NavBar-title'>{this.title}</h1>
```
Comment (dnastrain) [inline src/components/NavBar/NavBar.js]: iirc, Hao said this could also be {this.props.title}

**Review by dnastrain — DISMISSED** [review]

File: src/components/NavBar/NavBar.js (hunk 1 above)
Comment (dwcares) [reply src/components/NavBar/NavBar.js]: fixed

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './NavBar.css'
+
+class NavBar extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {value: ''}
+    this.data = [{key: 0, value: 'coconut'}, {key: 1, value: 'banana'}, {key: 2, value: 'lime'}]
+    this.handleChange = this.handleChange.bind(this)
```
Comment (howlowck) [inline src/components/NavBar/NavBar.js]: What is this line doing? @dwcares

**Review by ScruffyFurn — DISMISSED** [review]

**Review by howlowck — CHANGES_REQUESTED** [review]

File: src/components/NavBar/NavBar.js (hunk 2 above)
Comment (dwcares) [reply src/components/NavBar/NavBar.js]: it's not needed. fixed!

**Comment by saroup** [comment]
@  can you make sure you pull from dev. It looks like travis is not able to run on this pull request

**Review by howlowck — APPROVED** [review]
Looks Good!

**Pull request merged**